android/src/main/java/com/gettipsi/stripe/Errors.java

```diff
@@ -27,11 +32,15 @@
     exceptionNameToErrorCode.put("APIException", "api");
   }
 
-  static String toErrorCode(@NonNull Exception exception) {
+  public static String toErrorCode(@NonNull Exception exception) {
     ArgCheck.nonNull(exception);
     String simpleName = exception.getClass().getSimpleName();
     String errorCode = exceptionNameToErrorCode.get(simpleName);
-    ArgCheck.nonNull(errorCode, simpleName);
+
+    if (errorCode == null) {
+      errorCode = simpleName;
+    }
+//    ArgCheck.nonNull(errorCode, simpleName);
```

Comment:
Seems like it would be better just to delete this. This should definitely be non-null at this point as .getSimpleName will always return non-null.